src/pages/Review/Review.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import Total from './Total';
+import { withRouter } from 'react-router-dom';
+import ReviewList from './ReviewList';
+import { BASE_URL } from '../../config.js';
+import './Review.scss';
+
+class Review extends Component {
+  constructor() {
+    super();
+    this.state = {
+      reviewList: [],
+      itemId: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch(`${BASE_URL}/products/${this.props.match.params.id}`)
+      .then(data => data.json())
+      .then(data => {
+        console.log(data.RESULT);
+        this.setState({
+          reviewList: data.RESULT.review,
+          itemId: this.props.match.params.id,
+        });
+      });
+  }
+
+  reviewWriteBtn = () => {
+    fetch(`${BASE_URL}/products/${this.props.match.params.id}`, {
+      method: 'POST',
+      body: JSON.stringify({
+        //token: localStorage.getItem('TOKKEN'),
+        itemId: this.state.itemId,
+      }),
+      headers: {
+        Authorization: localStorage.getItem('TOKKEN'),
```

Comment:
```suggestion
        Authorization: localStorage.getItem('TOKEN'),
```
이런 오타들이 쌓이면 나중에 디버깅 하기 어려워지기 때문에 제때 수정해주는 것이 좋습니다

Reply:
백엔드분들에게 말씀해서 수정완료했습니다~~!!!